src/app/api/mock/hibp/range/search/[hashPrefix]/route.ts

```diff
@@ -0,0 +1,35 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import { NextResponse } from "next/server";
+import { logger } from "../../../../../../functions/server/logging";
+import { getSha1 } from "../../../../../../../utils/fxa";
+import fakeBreaches from "../../../data/fakeBreaches.json";
+import type { BinaryLike } from "crypto";
+import { errorIfProduction } from "../../../../utils/errorThrower";
+
+type BreachedAccountResponse = {
+  hashSuffix: string;
+  websites: string[];
+}[];
+
+export function GET() {
+  const prodError = errorIfProduction();
+  if (prodError) return prodError;
+
+  // Mock data for test email, can be randomized
+  const userEmail = process.env.E2E_TEST_ACCOUNT_EMAIL;
+  //TODO: getServerSession doesn't work here for some reason
```

Comment:
This would be expected if e.g. this is called by `fetch` without passing credentials such as a `Bearer` token.